binanceAlarmProject/settings.py

```diff
@@ -124,3 +125,5 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+AUTH_USER_MODEL = 'trading.PhoneNumber'
```

Comment:
I don't fully understand the question. Can you please explain?

Reply:
Where AUTH_USER_MODEL used? Does it used in the admin page login flow? If yes, will it change admin page login form?

Reply:
Yes, it is used in the admin page login flow. This will use the phone number model to register. The user will register with a phone number, not an email

Reply:
It is very confusing to have a user model named PhoneNumber. Judging from naming, I'd expect either User model to have a phone_number field, either if I even have a PhoneNumber model, I'd expect it to have a FK to a user model. Please make a PhoneNumber an independent model.

Reply:
Naming should always give a quick hint of what the variable is responsible for. This naming does not give a hint, but what's worse - it gives a desinformation. It's like you go to the shop, they say 250 UAH, but they actually want 250 $.